addon/modifiers/container-query.ts

```diff
@@ -154,14 +158,14 @@ export default class ContainerQueryModifier<
 
     for (const [featureName, meetsFeature] of Object.entries(
       this.queryResults
-    )) {
+    ) as ObjectEntries<QueryResults<T>>) {
       if (!meetsFeature) {
         continue;
       }
 
       const dataAttribute = prefix
-        ? `data-${prefix}-${featureName}`
-        : `data-${featureName}`;
+        ? `data-${prefix}-${String(featureName)}`
+        : `data-${String(featureName)}`;
```

Comment:
Because the generic type `T` extends `IndexSignatureParameter` (which is defined to be `string | number | symbol`), TypeScript threw an error and asked that `String()` is used to convert `featureName` in case it is a symbol.